### ISSUE TITLE
Possiblity to train with extremely anisotrpic images

### ISSUE THREAD

**Issue by CinKKKyo** [ai-med/Vox2Cortex#14]
Hi, I saw the size of the image the paper used is (128, 144, 128). While in my own dataset, the size of the image is always extremely anisotropic(such as (192, 192, 32)). Is it possible to use Vox2Cortex to train on extremely anisotropic data without any modification directly?

**Comment by fabibo3** [comment]
I think, in general, this should be possible. However, it might be necessary to adapt a couple of parameters in the implementation (PATCH_SIZE, SELECT_PATCH_SIZE) and potentially tune loss function weights.

**Comment by AndyCA111** [comment]
Hi, do you have a pre-trained model

**Comment by fabibo3** [comment]
We just uploaded a model checkpoint, please see `experiments/public_model` and the updated README.

**Comment by fabibo3** [comment]
I'll close this for now, feel free to re-open if necessary.
